packages/shared/src/lib/func.ts

```diff
@@ -144,3 +152,10 @@ export const initReactModal = ({
 
   globalThis.reactModalInit = true;
 };
+
+export const initApp = (): void => {
```

Comment:
@sshanzel The android app does not set any permanent indicator that it is an app as of yet like the ios PWA does. So until we implement something more robust like setting the user agent on webview launch, this will have to do!